examples/cookbook/architecture/command/lib/result.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Utility class to wrap result data
```

Comment:
Consider fleshing out this doc comment. It could explain that it encapsulates a successful or error result. It could also explain what `T` is, etc.

Reply:
Improved this doc